packages/ipfs-core-utils/test/files/normalise-input.spec.js

```diff
@@ -35,7 +36,12 @@ async function verifyNormalisation (input) {
   let content = input[0].content
 
   if (isBrowser || isWebWorker || isElectronRenderer) {
-    expect(content).to.be.an.instanceOf(Blob)
+    try {
```

Comment:
`File` extends `Blob` so under what conditions would we need to fall back here?

Reply:
`instanceOf` is tricky. 
File https://github.com/web-std/file/blob/default/src/lib.js#L6 extends from Blob as you mention, and Blob is https://github.com/web-std/io/blob/main/blob/src/lib.js (so the same as here).

But, somehow this will not work with instanceOf Blob, and works with `@web-std/blob`. So, if you create a Blob and a @web-std/blob, their instanceOf are not compatible...

Reply:
This sounds like `@web-std/blob` needs to return the native implementation if it's available?

Reply:
Also of note here is that node 16 has a [Blob](https://nodejs.org/api/buffer.html#buffer_class_blob) implementation.

Reply:
> This sounds like @web-std/blob needs to return the native implementation if it's available?

It does: https://github.com/web-std/io/blob/main/blob/src/lib.js

but instanceOf returns false 🤷🏼‍♂️

Reply:
> but instanceOf returns false 🤷🏼‍♂️

Then it can't be, otherwise instanceOf would return true?

Reply:
I changed it now to Blob to be:

```js
const { Blob } = require('@web-std/blob')
```

instead of 

```js
const { Blob } = globalThis
```

It works this way, but then other tests fail (which probably use the native Blob 🤷🏼‍♂️

Reply:
reverting this, as this is even more problematic

Reply:
So, when creating a https://github.com/web-std/io/tree/main/file in the browser, it will not be an instance of `Blob`, but it will be of https://github.com/web-std/io/tree/main/blob. This is really weird, given that ws Blob is really [globalThis.Blob](https://github.com/web-std/io/blob/main/blob/src/lib.js#L3).

This [codepen](https://codepen.io/vascosantos/pen/bGWrqMx?editors=0011) shows that a browser File should be instance of both Blob and ws Blob.

My only assumption for this to not be the case in the tests here, is that somehow playwright has a different globalThis scope in the test that makes them being different... @hugomrdias is this reasonable? do you have any clues?

Reply:
Now `@web-std/file` uses the Node.js version with Eletron renderer 😡